handler/core/src/main/java/io/knotx/fragments/task/factory/LogLevelConfig.java

```diff
@@ -25,6 +27,10 @@
 
   private String logLevel;
 
+  public LogLevelConfig() {
+    logLevel = ERROR.getLevel();
```

Comment:
it causes test failing. leave as it is.

Because this logic
```
  private void override(JsonObject json, String globalLogLevel) {
    LogLevelConfig logLevelConfig = new LogLevelConfig(json);
    if (StringUtils.isBlank(logLevelConfig.getLogLevel())) {
      json.mergeIn(logLevelConfig.setLogLevel(globalLogLevel).toJson());
    }
  }
```